packages/host/app/components/operator-mode/overlays.gts

```diff
@@ -401,12 +382,23 @@ export default class OperatorModeOverlays extends Component<Signature> {
       renderedCard.element.addEventListener(
         'mouseenter',
         // eslint-disable-next-line ember/no-side-effects
-        (_e: MouseEvent) => this.setCurrentlyHoveredCard(renderedCard),
+        (_ev: MouseEvent) => {
+          if (this.currentlyHoveredCard === renderedCard) {
+            return;
+          }
+          this.setCurrentlyHoveredCard(renderedCard);
+        },
       );
       renderedCard.element.addEventListener(
         'mouseleave',
         // eslint-disable-next-line ember/no-side-effects
-        (_e: MouseEvent) => this.setCurrentlyHoveredCard(null),
+        (ev: MouseEvent) => {
+          let relatedTarget = ev.relatedTarget as HTMLElement;
+          if (relatedTarget?.closest?.('.actions-overlay')) {
+            return;
+          }
+          this.setCurrentlyHoveredCard(null);
+        },
```

Comment:
I see the listener implementation already existed but is this a place for a modifier?

Reply:
Maybe. It's awkward, though, because we are doing this at a distance from the corresponding ember component